### ISSUE TITLE
Bundle for cell

### ISSUE THREAD

**Issue by ManWithBear** [Showmax/SHMTableView#2]
Is here any reasons not to use `Bundle(for: cell.T)` as default value?

**Comment by LukasKorba** [comment]
you mean `self.bundle = Bundle(for: cell.self)`?

**Comment by ManWithBear** [comment]
Yes. Only reason I see, if you want to use cell with nib from different bundle (pretty rare usecase).
